src/types/endpoints/service_endpoint.go

```diff
@@ -77,6 +79,19 @@ func (s *ServiceEndpoint) getPublicEndpoints() map[string]string {
 	return endpoints
 }
 
+func (s *ServiceEndpoint) getWorkerEndpoints() map[string]string {
+	host := fmt.Sprintf("%s_HOST", toConstantCase(s.ServiceRole))
+	endpoints := map[string]string{}
+	if s.ContainerPort != "" {
+		if s.BuildMode.Type == types.BuildModeTypeLocal {
+			endpoints[host] = fmt.Sprintf("http://%s:%s", s.ServiceRole, s.ContainerPort)
```

Comment:
I don't think this should have http at the beginning. That may not be the protocol